eo-runtime/src/test/java/EOorg/EOeolang/EOcageTest.java

```diff
@@ -255,6 +256,16 @@ void writesBoundedCopyOfTheSameBase() {
         );
     }
 
+    @Test
+    void throwsExceptionIfRecursion() {
+        final Phi cage = new EOcage(Phi.Φ);
+        writeTo(cage, cage);
+        Assertions.assertThrows(
```

Comment:
@levBagryansky Could you add a message to the assertion which will help to understand your expectations, please?
Why it's important: https://www.yegor256.com/2023/01/19/layout-of-tests.html#assertions

Reply:
@volodya-lombrozo added